<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageAlbum</class>
 <widget class="QWidget" name="ImageAlbum">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>400</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>400</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>600</width>
         <height>400</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>100</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>600</width>
         <height>80</height>
        </size>
       </property>
       <property name="flow">
        <enum>QListView::LeftToRight</enum>
       </property>
       <property name="resizeMode">
        <enum>QListView::Adjust</enum>
       </property>
       <property name="viewMode">
        <enum>QListView::IconMode</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QToolButton" name="ZoomIn">
         <property name="text">
          <string>줌인</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="ZoomOut">
         <property name="text">
          <string>줌아웃</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="LeftRotate">
         <property name="text">
          <string>좌회전</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="RightRotate">
         <property name="text">
          <string>우회전</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>밝기</string>
         </property>
         <property name="buddy">
          <cstring>horizontalSlider</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="horizontalSlider">
         <property name="minimum">
          <number>-70</number>
         </property>
         <property name="maximum">
          <number>70</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QToolButton" name="OrigImage">
         <property name="text">
          <string>원본</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QToolButton" name="Brush">
         <property name="text">
          <string>브러쉬</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QToolButton" name="Gamma">
         <property name="text">
          <string>감마</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QToolButton" name="Sobel">
         <property name="text">
          <string>소벨</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QToolButton" name="VReverse">
         <property name="text">
          <string>좌우 반전</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QToolButton" name="HReverse">
         <property name="text">
          <string>상하 반전</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
